test_scripts/Resumption/InteriorVehicleData/025_InteriorVD_resumption_double_subscription_without_moduleId_unexpected_disconnect.lua

```diff
@@ -0,0 +1,94 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL sends GetInteriorVehicleData response with WARNINGS result code
+--  in case of double subscription with default moduleId after successful resuming on transport disconnect
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to all modules via GetInteriorVehicleData(moduleType)
+-- 4. Unexpected disconnect and reconnect are performed
+-- 5. App reregisters with actual HashId after unexpected disconnect
+-- 6. HMI responds with SUCCESS resultCode to all requests from SDL
+--
+-- Sequence:
+-- 1. GetInteriorVD(subscribe = true, module_1) is requested
+-- SDL does:
+-- - a. does not send GetInteriorVD(subscribe = true, module_1, default moduleId) request to HMI
+-- - b. send GetInteriorVD ("WARNINGS") response to mobile App
+-- - c. does not send OnHashChange notification to mobile app by receiving
+--    GetInteriorVD(subscribe = true, module_1, default moduleId)
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/Resumption/InteriorVehicleData/commonResumptionsInteriorVD')
+
+--[[ Local Variables ]]
+local withoutModuleId = nil
+local notExpectNotif = 0
+local expectNotif = 1
+local isSubscribed = true
+local isCached = true
+local isNotCached = false
+local appSessionId = 1
+local resultCode = "WARNING"
+
+--[[ Local Functions ]]
+local function checkResumptionData()
+  local defaultModuleId = 1
+  local actualModules = { }
+  local expectedModules = { }
+
+  for key, moduleType in pairs(common.modules) do
+    expectedModules[key] = {
+      moduleType = moduleType,
+      moduleId = common.getModuleId(moduleType, defaultModuleId)
+    }
+  end
+
+  common.getHMIConnection():ExpectRequest("RC.GetInteriorVehicleData")
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {
+        moduleData = common.getActualModuleIVData(data.params.moduleType, data.params.moduleId), isSubscribed = true })
+    end)
+  :ValidIf(function(exp, data)
+    actualModules[exp.occurences] = {
+      moduleType = data.params.moduleType,
+      moduleId = data.params.moduleId
+    }
+    if exp.occurences == #common.modules then
```

Comment:
@VjKlepikov This check will fail in all cases except last one because each request has different `exp.occurences` value.
Please fix the check.

Reply:
@VjKlepikov It was discussed. No changes required.